examples/templates/azure-linux/main.tf

```diff
@@ -2,73 +2,215 @@ terraform {
   required_providers {
     coder = {
       source  = "coder/coder"
-      version = "~> 0.6.12"
+      version = "~> 0.6.17"
     }
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "=3.0.0"
+      version = "~>3.47.0"
     }
   }
 }
 
-variable "location" {
+data "coder_parameter" "location" {
+  name        = "Location"
   description = "What location should your workspace live in?"
   default     = "eastus"
-  validation {
-    condition = contains([
-      "eastus",
-      "southcentralus",
-      "westus2",
-      "australiaeast",
-      "southeastasia",
-      "northeurope",
-      "westeurope",
-      "centralindia",
-      "eastasia",
-      "japaneast",
-      "brazilsouth",
-      "asia",
-      "asiapacific",
-      "australia",
-      "brazil",
-      "india",
-      "japan",
-      "southafrica",
-      "switzerland",
-      "uae",
-    ], var.location)
-    error_message = "Invalid location!"
-  }
-}
-
-variable "instance_type" {
+  icon        = "/emojis/1f310.png"
```

Comment:
Icons seem to be broken here.

<img width="584" alt="Screenshot 2023-03-15 at 13 21 39" src="https://user-images.githubusercontent.com/14044910/225307669-e9aea36c-c33c-4204-953f-5cb60e1c908e.png">

Reply:
The value is corect and it should work. This should display a wireframe of globe.

Reply:
it was a typo. the correct name of the icons directory is `icon` not `icons`. Sorry.